tools/merge_master.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""In a continuous integration environment, merge the master branch into the
+current branch before build & test"""
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+
+import github_update
+
+logger = logging.getLogger(__name__)
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO, format="%(asctime)s %(name)s %(message)s")
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        description="merge a branch to the current branch for continuous integration goals")
+    parser.add_argument("branch",
+            nargs="?",
+            help="Branch to merge",
+            default="master")
+    return parser.parse_args(args)
+
```

Comment:
I personally don't really understand this style suggestion in PEP8 or in genearal.  Meaningless vertical whitespace doesn't help readability, and reduces the amount of code viewable.  What am I missing?  

Done anyway.